mirror-service/src/main/java/org/hyades/dto/OsvDto.java

```diff
@@ -0,0 +1,212 @@
+package org.hyades.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+import org.cyclonedx.model.ExternalReference;
+import org.cyclonedx.model.OrganizationalContact;
+import org.cyclonedx.model.vulnerability.Vulnerability;
+
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import static org.hyades.commonutil.JsonUtil.jsonStringToTimestamp;
+import static org.hyades.model.Vulnerability.Source.GITHUB;
+import static org.hyades.model.Vulnerability.Source.NVD;
+import static org.hyades.model.Vulnerability.Source.OSV;
+
+@RegisterForReflection
+public class OsvDto implements Serializable {
+    private String id;
+    private String summary;
+    private String details;
+    private String published;
+    private String modified;
+    private String withdrawn;
+    private List<String> aliases;
+    private List<Credit> credits;
+    private List<Reference> references;
+
+    @JsonProperty(("database_specific"))
```

Comment:
can this be a more indicative name? not. very clear

Reply:
I can link Osv spec doc. https://ossf.github.io/osv-schema/#database_specific-field 
I thought about it but can't think of a better name. It really is database specific. Do you have another suggestion ?

Reply:
okay yeah maybe in a comment this documentation link would be useful